client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -40,7 +40,7 @@ import { getUser } from 'state/users/selectors';
 import { managePurchase } from '../paths';
 import AutoRenewToggle from './auto-renew-toggle';
 import PaymentLogo from 'components/payment-logo';
-import { CALYPSO_CONTACT } from 'lib/url/support';
+import { JETPACK_SUPPORT } from 'lib/url/support';
```

Comment:
I can see this is getting Jetpackified; but could anyone confirm that this will happen only for Jetpack sites? Can't it happen for simple sites? What about atomic sites?

Reply:
Is it even possible that a simple or an atomic site loses a connection with WPCOM and this message would appear for a user? Maybe @enejb could chime in to answer since I don't really know.

Reply:
I think @tyxla  make a good point. 

I think it would be good to check what type of plan/product the site has. 
And display a different message based on that. 

I can imagine a user buying a simple site product but be removed from the site. Hence the site not being there any more for that user. In that case they need to transfer the site. 

For Jetpack sites the main issue is probably some sort of disconnection not them being kicked off the site. Since that is a more common error. 

Can we do a check for what kinds of plan/product the site has before showing them a new message?

Reply:
@tyxla @enejb I've updated the PR so that now the new message is shown only for Jetpack sites. In any other (edge) case, the current message will be shown (i.e. contact support).

I've updated the method name and left the CSS selector name change so that they make more sense in various contexts.

Please have another look.